src/garrysmod/gamemodes/zombiesurvival/gamemode/languages/english.lua

```diff
@@ -537,3 +537,36 @@ LANGUAGE.help_cont_being_a_zombie					= [[<p>Tips for this section:
 ]]
 
 -- Place any custom stuff below here...
+LANGUAGE.vote_for_the_next_map = "Vote for the next map"
+LANGUAGE.map_vote_ends_in = "Map vote ends in"
+LANGUAGE.youre_playing_on = "You're playing on %s"
+LANGUAGE.map_changes_in_or_in = "Map changes in %d rounds or in %s"
+
+LANGUAGE.zs_18_v2a = "18"
+LANGUAGE.zs_abandoned_mall_starbucks_v2 = "Abandoned Mall"
+LANGUAGE.zs_adrift_v2 = "Adrift"
+LANGUAGE.zs_bar_v5 = "Bar"
+LANGUAGE.zs_bdf_b3 = "Butchers Den"
+LANGUAGE.zs_cabin_v7 = "Cabin"
+LANGUAGE.zs_closure_v2 = "Closure"
+LANGUAGE.zs_coast_of_konigsberg_v5c = "Coast of Koningsberg"
+LANGUAGE.zs_comfy_v1a = "Comfy"
+LANGUAGE.zs_cursed_woods_v2 = "Cursed Woods"
+LANGUAGE.zs_danzig_docks_v4 = "Danzig Docks"
+LANGUAGE.zs_daycare_v7 = "Daycare"
+LANGUAGE.zs_development_v2 = "Development"
+LANGUAGE.zs_dezecrated = "Dezecrated"
+LANGUAGE.zs_downtown_v1 = "Downtown"
+LANGUAGE.zs_draynor_b2 = "Draynor"
+LANGUAGE.zs_dryfield_2018 = "Dryfield"
+LANGUAGE.zs_erebus_v2 = "Erebus"
+LANGUAGE.zs_horse_city_v5 = "Horse City"
+LANGUAGE.zs_limansk_v2 = "Limansk"
+LANGUAGE.zs_lunar_kingdom_v1c = "Lunar Kingdom"
+LANGUAGE.zs_nipperhouse_2019 = "Nipper House"
+LANGUAGE.zs_office_apocalypse_v1 = "Office Apocalypse"
+LANGUAGE.zs_palace_v2 = "Palace"
+LANGUAGE.zs_power_cliff_2019 = "Power Cliff"
+LANGUAGE.zs_purple_v1 = "Purple"
+LANGUAGE.zs_stoyachava = "Stoyachava"
+LANGUAGE.zs_winter_solstice_v2 = "Winter Solstice"
```

Comment:
It's probably possible to compute these names from the bsp name? Standard regex to extract the map name that should cover ~99% of cases would be something akin to: `/(?:zs_)(.*?)(?=(?:_(?:a|b|v)[0-9])|$)/i`, although I haven't tried to port that to Lua pattern matching. 

Otherwise adding new maps will require this list to be manually maintained as well

Reply:
I plugged this into Regex101. I'd argue there are too many edge cases for this to work (a lot larger than 1%).

![image](https://user-images.githubusercontent.com/1774125/92408774-81ee4880-f10c-11ea-8430-c863709d75b9.png)

I'm aware that adding new maps will require this list to be maintained. We will still have to add metadata in the form of images for every map, so I don't think there's a way out of this unless we develop a mechanism to read the metadata from S3.

I think that'd be out of scope from this issue, but something worth exploring down the road.

Reply:
Suffixes that aren't prefixed with _a, _b or _x are rather uncommon, but that's fair; I think our maplist is skewed against common naming conventions. 
I did consider maps ending in _<year>; I'd argue they should have the year included anyway, as they are (in theory) not the original and it might be confusing that 'Dryfield' on the mapvote isn't actually 'DryField' but 'DryField_2018'